python/pyproject.toml

```diff
@@ -20,4 +20,30 @@ requires = [
     "setuptools>=42",
     "wheel"
 ]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 130
+target-version = ['py38']
+
+[tool.isort]
+src_paths = ["python/src/", "python/tests/"]
```

Comment:
Should this be relative the `pyproject.toml` file i.e. `["src/", "tests/"]` , like we have it configured for `tool.coverage.run` below.

Reply:
You're right, thanks!